addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -30,8 +30,10 @@ pscanrules.authenticationcredentialscaptured.soln = Use HTTPS, and use a secure
 
 pscanrules.bigredirects.desc = The server has responded with a redirect that seems to provide a large response. This may indicate that although the server sent a redirect it also responded with body content (which may include sensitive details, PII, etc.).
 pscanrules.bigredirects.extrainfo = Location header URI length: {0} [{1}].\nPredicted response size: {2}.\nResponse Body Length: {3}.
+pscanrules.bigredirects.multi.desc = The server has responded with a redirect that seems to provide multiple links. This may indicate that although the server sent a redirect it also responded with body content (which may include sensitive details, PII, etc.).
+pscanrules.bigredirects.multi.extrainfo = The response contained {0} occurrences of "HREF".
+pscanrules.bigredirects.multi.name = Strange Redirect Detected (Potential Sensitive Information Leak)
```

Comment:
Maybe `Multiple HREFs Redirect Detected`.

Reply:
Oh crap, will tweak in a bit, we can finish tomorrow. Thanks for your review!